include/openssl/crypto.h

```diff
@@ -287,6 +287,7 @@ void OPENSSL_cleanse(void *ptr, size_t len);
         CRYPTO_mem_debug_push(info, OPENSSL_FILE, OPENSSL_LINE)
 #  define OPENSSL_mem_debug_pop() \
         CRYPTO_mem_debug_pop()
+void CRYPTO_malloc_setup_failures(void);
```

Comment:
I'm dubious about having this publically declared.  Considering it's called by the init code anyway, *we* don't need it to be public.